app/views/pages/admin_dashboard.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Admin dashboard</h1>
+
+
+<div class="container">
+  <br>
+  <h1>Admin dashboard</h1>
+  <br>
+  <h3>Welcome <%= current_user.first_name%>! </h3>
+  <%= link_to "Your students", students_path%>
+  <br>
+  <br>
+  <%= link_to "Your teachers", teachers_path%>
+  <%= link_to "Add teacher", teacher_path%>
+  <br>
+  <br>
+  <%= link_to "Your subscriptions", studio_subscriptions_path(current_user.studio_id)%>
```

Comment:
do we have to pass current_user.studio_id or is current_user.studio enough? I have never seen current_user.studio_id before

Reply:
i guess current_user.studio is enough